FirebaseRemoteConfig/Sources/RCNConfigDBManager.m

```diff
@@ -798,6 +845,79 @@ - (void)loadExperimentWithCompletionHandler:(RCNDBCompletion)handler {
   return results;
 }
 
+- (void)loadPersonalizationWithCompletionHandler:(RCNDBLoadCompletion)handler {
+  __weak RCNConfigDBManager *weakSelf = self;
+  dispatch_async(_databaseOperationQueue, ^{
+    RCNConfigDBManager *strongSelf = weakSelf;
+    if (!strongSelf) {
+      return;
+    }
+
+    NSDictionary *activePersonalization;
+    NSData *personalizationResult = [strongSelf loadPersonalizationTableFromKey:RCNDBSourceActive];
+    // There should be only one entry for Personalization metadata.
+    if (personalizationResult) {
+      NSError *error;
+      activePersonalization = [NSJSONSerialization JSONObjectWithData:personalizationResult
+                                                              options:0
+                                                                error:&error];
+    }
+    if (!activePersonalization) {
+      activePersonalization = [[NSMutableDictionary alloc] init];
+    }
+
+    NSDictionary *fetchedPersonalization;
+    personalizationResult = [strongSelf loadPersonalizationTableFromKey:RCNDBSourceFetched];
+    // There should be only one entry for Personalization metadata.
+    if (personalizationResult) {
+      NSError *error;
+      fetchedPersonalization = [NSJSONSerialization JSONObjectWithData:personalizationResult
+                                                               options:0
+                                                                 error:&error];
+    }
+    if (!fetchedPersonalization) {
+      fetchedPersonalization = [[NSMutableDictionary alloc] init];
+    }
+
+    if (handler) {
+      dispatch_async(dispatch_get_main_queue(), ^{
+        handler(YES, activePersonalization, fetchedPersonalization, nil);
```

Comment:
I think the handler definition expects `fetched` and then `active`

Reply:
LOL, too much copying and pasting.